service.libraryautoupdate/resources/lib/relativedelta.py

```diff
@@ -0,0 +1,430 @@
+"""
```

Comment:
Better as a script.module?

Reply:
Different license than the add-on too...

Reply:
I'll move this to a module. Been part of the addon for so long I've never really looked at the source. Probably been in there since 2012 or earlier.

Reply:
Thanks for pointing this out. In researching the source for this file I found it's part of an existing module script.module.dateutil. I added a dependency for that addon and removed the file from this one.